examples/tornjak/values.yaml

```diff
@@ -0,0 +1,47 @@
+spire-server:
+  tornjak:
+    enabled: true
+    image:
+      registry: ghcr.io
+      repository: spiffe/tornjak-be
+      pullPolicy: IfNotPresent
+      version: "v1.0.2"
+    config:
+      backend:
+        apiServerURL: "http://localhost:10000/"
+        dataStore:
+          driver: "sqlite3"
+          file: "/run/spire/data/tornjak.sqlite3"
+
+tornjak-frontend:
+  enabled: true
+  image:
+    registry: ghcr.io
+    repository: spiffe/tornjak-fe
+    pullPolicy: IfNotPresent
+    version: "v1.0.2"
+  imagePullSecrets: []
+  nameOverride: ""
+  namespaceOverride: ""
+  fullnameOverride: ""
+  serviceAccount:
+    create: true
+    annotations: {}
+    name: ""
+  podSecurityContext:
+    runAsUser: 1000
+    runAsGroup: 1000
+    fsGroup: 1000
+    fsGroupChangePolicy: OnRootMismatch
+  securityContext:
+    allowPrivilegeEscalation: false
+    runAsNonRoot: true
+    readOnlyRootFilesystem: false
+    capabilities:
+      drop: [ALL]
+    seccompProfile:
+      type: RuntimeDefault
```

Comment:
Thinking this stuff should be put in the production examples values file. Then this values file becomes more about turning on/cofiguraing tornjak, while the other one is about productionizing all the settings for the possible features.

Reply:
So about that, when I move the `podSecurityContext` and `securityContext` values to _examples/production/values.yaml_ they got rendered as: 
```yaml
     securityContext:
         {}
```         
 All the other values work fine. I need to figure out why. Perhaps it has something to do how we format the values in the template?:
 ```yaml
       securityContext:
        {{- toYaml .Values.podSecurityContext | nindent 8 }}   
 ```

Reply:
I tried again and it seems to work the second time around.